src/main/resources/de/qabel/desktop/ui/sync/item/syncitem.fxml

```diff
@@ -4,171 +4,190 @@
 <?import javafx.scene.control.*?>
 <?import javafx.scene.image.*?>
 <?import javafx.scene.layout.*?>
-<?import javafx.scene.text.*?>
-<BorderPane fx:id="syncItemRoot" styleClass="syncItem" prefHeight="130.0" prefWidth="600.0" style="-fx-border-color: darkgrey; -fx-border-width: 1 0 0 0;" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1" fx:controller="de.qabel.desktop.ui.sync.item.SyncItemController">
-            <left>
-                <VBox fx:id="avatarContainer" BorderPane.alignment="CENTER" />
-            </left>
-            <padding>
-                <Insets bottom="10.0" left="10.0" right="10.0" top="10.0" />
-            </padding>
+<?import javafx.scene.text.Font?>
+<BorderPane fx:id="syncItemRoot" styleClass="syncItem" prefHeight="130.0" prefWidth="600.0"
+            xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1"
+            fx:controller="de.qabel.desktop.ui.sync.item.SyncItemController"
+            stylesheets="@../../../../../../main.css">
+    <padding>
+        <Insets bottom="10.0" left="10.0" right="10.0" top="10.0" />
+    </padding>
+    <center>
+        <BorderPane prefHeight="200.0" prefWidth="200.0" BorderPane.alignment="CENTER">
             <center>
-                <BorderPane prefHeight="200.0" prefWidth="200.0" BorderPane.alignment="CENTER">
-                    <center>
-                        <VBox BorderPane.alignment="CENTER" fx:id="statusContentPane">
+                <VBox fx:id="statusContentPane" BorderPane.alignment="CENTER">
+                    <children>
+                        <VBox prefHeight="200.0" prefWidth="100.0" style="-fx-spacing: 3;">
+                            <padding>
+                                <Insets bottom="5.0" left="5.0" right="5.0" top="5.0" />
+                            </padding>
                             <children>
-                                <VBox prefHeight="200.0" prefWidth="100.0" style="-fx-spacing: 3;">
```

Comment:
because i reformatted this file on purpose it was not niceley readable for me.
wouldve be better if i seperated it in 2 seperate commits, one the real "just add stuff" and one make it pretty :)